dataclients/kubernetes/kube_test.go

```diff
@@ -3639,3 +3640,147 @@ func TestCreateEastWestRouteOverwriteDomain(t *testing.T) {
 		})
 	}
 }
+
+func TestSkipperDefaultFilters(t *testing.T) {
+	api := newTestAPI(t, nil, &ingressList{})
+	defer api.Close()
+
+	t.Run("check routes are created if default filters dir is not set", func(t *testing.T) {
+		api.services = testServices()
+		api.ingresses.Items = testIngresses()
+
+		dc, err := New(Options{ // DefaultFiltersDir setting is not set
+			KubernetesURL: api.server.URL,
+		})
+		if err != nil {
+			t.Error(err)
+		}
+
+		defer dc.Close()
+
+		r, err := dc.LoadAll()
+
+		assert.Nil(t, err)
```

Comment:
Please do not use `assert`, we would like to drop the library in the future.

Reply:
Interesting. Why?

Reply:
Because it adds maintenance work for using dependencies and bloat the binary and compile time